_pytest/test_multitenancy.py

```diff
@@ -119,19 +115,19 @@ def test_post_parse_invalid_model(client, response_test):
     assert response.json.get("error").startswith(response_test.expected_response["error"])
 
 
-if __name__ == '__main__':
```

Comment:
I call the train() function each time because I had an issue where the `test_models` trained with python 2 where not loadable with python 3 and vice versa because of pickle (the issue was more specifically related to [joblib](https://pythonhosted.org/joblib/persistence.html#compatibility-across-python-versions))